packages/blocks/src/__internal__/utils/selection.ts

```diff
@@ -22,6 +22,99 @@ const notStrictCharacterReg = /[^\p{Alpha}\p{M}\p{Nd}\p{Pc}\p{Join_C}]/u;
 const notStrictCharacterAndSpaceReg =
   /[^\p{Alpha}\p{M}\p{Nd}\p{Pc}\p{Join_C}\s]/u;
 
+// function fixCurrentRangeToText(
```

Comment:
We shouldn't checkin large batch of commented code in the codebase (unless it's a WIP PR), it won't be maintained by others and will be staled.